src/XrmDefinitelyTyped/XrmDefinitelyTyped.fsproj

```diff
@@ -81,21 +81,23 @@
     <Compile Include="Domain.fs" />
     <Compile Include="Utility.fs" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_6-.d.ts" />
-    <EmbeddedResource Include="Resources\Extensions\xrm_ext_-8.d.ts" />
+    <EmbeddedResource Include="Resources\Extensions\xrm_ext_-6.d.ts" />
+    <EmbeddedResource Include="Resources\Extensions\xrm_ext_7-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_7.1-.d.ts" />
+    <EmbeddedResource Include="Resources\Extensions\xrm_ext_7.1-9-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_8-.d.ts" />
+    <EmbeddedResource Include="Resources\Extensions\xrm_ext_-8.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_8.2-.d.ts" />
-    <EmbeddedResource Include="Resources\Extensions\xrm_ext_7-.d.ts" />
-    <EmbeddedResource Include="Resources\Extensions\xrm_ext_-6.d.ts" />
-    <EmbeddedResource Include="Resources\Extensions\xrm_ext_-9-.d.ts" />
-    <EmbeddedResource Include="Resources\Extensions\xrm_ext_7.1-9-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_8.2-9-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_6-9-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_7-9-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_9-.d.ts" />
+    <EmbeddedResource Include="Resources\Extensions\xrm_ext_-9-.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\xrm_ext_-9.d.ts" />
     <EmbeddedResource Include="Resources\Extensions\dg.xrmquery.web_ext_9-.ts" />
     <EmbeddedResource Include="Resources\Extensions\dg.xrmquery.web_ext_9-.d.ts" />
+    <None Include="Resources\Extensions\xrm_ext_8-9.d.ts" />
```

Comment:
This seems to stick out. It might be important that the resource is embedded, change it so it matches the others

Reply:
Yeah. The resources needs to be embedded as it pulls the extension files from the assembly on runtime. Otherwise the extension file makes no changes